### ISSUE TITLE
Invasion Point: Anihilator part 4 of the quest chain

### ISSUE THREAD

**Issue by Atheras** [FireFly-WoW/FireFly-IssueTracker#357]
**Description:**

The quest item doesnt seem to drop from Warbringer Arix' Amal, he should drop a key that i can use to disrupt the portal.

Quest is located in northern part of Hellfire Peninsula

**Comment by Demmye** [comment]
Indeed, it does not have 100% which it should in Wrath of the Lich King expansion.
